apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.tsx

```diff
@@ -244,34 +246,42 @@ const OperationD = (props: Props) => {
     : undefined;
   const editEthTx = useFeature("editEthTx");
   const editable = editEthTx?.enabled && isEditableOperation(mainAccount, operation);
-  const dispatch = useDispatch();
-  const handleOpenEditModal = useCallback(
-    (account, parentAccount, transactionRaw, transactionHash) => {
-      setDrawer(undefined);
-      if (subOperations.length > 0 && isToken) {
-        // if the operation is a token operation,(ERC-20 send), in order to speedup/cancel we need to find the subAccount
-        const opAccount = findSubAccountById(account, subOperations[0].accountId);
-        dispatch(
-          openModal("MODAL_EDIT_TRANSACTION", {
-            account: opAccount,
-            parentAccount: account,
-            transactionRaw,
-            transactionHash,
-          }),
-        );
-      } else {
-        dispatch(
-          openModal("MODAL_EDIT_TRANSACTION", {
-            account,
-            parentAccount,
-            transactionRaw,
-            transactionHash,
-          }),
-        );
-      }
-    },
-    [dispatch, isToken, subOperations],
-  );
+  // const dispatch = useDispatch();
+  // const handleOpenEditModal = useCallback(
+  //   (account, parentAccount, transactionRaw, transactionHash) => {
+  //     setDrawer(undefined);
+  //     if (subOperations.length > 0 && isToken) {
+  //       // if the operation is a token operation,(ERC-20 send), in order to speedup/cancel we need to find the subAccount
+  //       const opAccount = findSubAccountById(account, subOperations[0].accountId);
+  //       dispatch(
+  //         openModal("MODAL_EDIT_TRANSACTION", {
+  //           account: opAccount,
+  //           parentAccount: account,
+  //           transactionRaw,
+  //           transactionHash,
+  //         }),
+  //       );
+  //     } else {
+  //       dispatch(
+  //         openModal("MODAL_EDIT_TRANSACTION", {
+  //           account,
+  //           parentAccount,
+  //           transactionRaw,
+  //           transactionHash,
+  //         }),
+  //       );
+  //     }
+  //   },
+  //   [dispatch, isToken, subOperations],
+  // );
+
+  const handleOpenEditModal = (
+    _account: unknown,
+    _parentAccount: unknown,
+    _transactionRaw: unknown,
+    _transactionHash: unknown,
+  ) => {};
+
```

Comment:
line 402-423 in this file, we should remove the alert message for editable operation since we remove the edit tx feature.